packages/common/rollup-configs.js

```diff
@@ -14,7 +16,7 @@ const getExternalModules = (pkg) => [
 const getExternalModuleTester = (pkg) => {
   const externalModules = getExternalModules(pkg);
   const externalModuleTests = externalModules.map((module) => new RegExp(`^${module}(\\/.+)*$`));
-  return (source) => externalModuleTests.some((regexp) => regexp.test(source));
+  return (moduleID) => externalModuleTests.some((regexp) => regexp.test(moduleID));
```

Comment:
In Rollup's terminology, this is a "module ID" so I've renamed the argument to stay consistent with that.

FYI - webpack calls them "module requests" while ECMAscript spec calls them "module specifiers".

Wish that everyone just used the same naming :face_exhaling: